pages/roadmap-test/index.tsx

```diff
@@ -1,92 +0,0 @@
-import type { NextPage } from 'next';
```

Comment:
we should leave these. I was working on input to allow devs to encode json necessary to make debugging and development faster.

Reply:
Same as https://github.com/pln-planning-tools/Starmaps/pull/81#discussion_r1020592066